### DISCUSSION TITLE
i found a trojan listed in github download as clean, dunno where to post it

### DISCUSSION THREAD

**Discussion by bobekkpl** [community/community#142642] [New to GitHub]
https://github.com/austintools/SFVIP-Player

if u scroll down to download section where it says its clean file, Virustotal says it's full of trojans trojans oposite to zip u download from "code"menu 

sorry dunno where to post it 
![image](https://github.com/user-attachments/assets/5fb54a36-d012-46c1-85d0-64bdb2ea3c65)
![image](https://github.com/user-attachments/assets/8402c328-059a-4f90-bbe5-ab81f1029dbd)

**Comment by bst04** [comment]
Talk with GitHub

**Comment by ghostinhershell** [comment]
Hi @bobekkpl, 👋🏻 We really appreciate you flagging this! The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by sir-smokealot-420** [comment]
I realized the same yesterday after installing this player. My cpu temp was quite high idling and when opened task manager, temp went back to normal. Windows Defender was not able to detect this.
